pkg/ibmvpc/vpc.go

```diff
@@ -739,7 +739,7 @@ type TransitGateway struct {
 	// vpcs are the VPCs connected by a TGW
 	vpcs []*VPC
 
-	// list of transit connections of the tgw
+	// list of all transit connections (not only those relevant to <src, dst>)
```

Comment:
change comment ending `(not only those relevant to this TransitGateway)`

Reply:
done